app/controllers/application_controller.rb

```diff
@@ -4,16 +4,18 @@ class ApplicationController < ActionController::Base
 
   add_flash_types :success
 
+  # NOTE: once we have applications linked to each provider, this
+  # needs to be scoped to the currently signed in provider
   def current_crime_application
-    @current_crime_application ||= CrimeApplication.find_by(id: session[:crime_application_id])
+    @current_crime_application ||= CrimeApplication.find_by(id: params[:id])
```

Comment:
I guess as we build in the auth system we will want to scope this by the providers office? I imagine they individuals could swap work so it makes sense to see cases related to a given office, though will need to check first if this assumption is correct.

Reply:
Yes, there is a comment right above this method precisely for that 😄